launch/launch/logging/__init__.py

```diff
@@ -340,7 +341,7 @@ def _normalize_output_configuration(config):
     normalized_config = {
         'both': set(), 'stdout': set(), 'stderr': set()
     }
-    if isinstance(config, str):
+    if isinstance(config, Text):
```

Comment:
Can you please explain why you made this change?  Text is a `typing` type, not a regular Python type, so I'm not sure that this is necessary.

Reply:
In trying to answer your question, I tried again to reproduce the issue and found that I couldn't. I suspect that I set `$OVERRIDE_LAUNCH_PROCESS_OUTPUT` but did not export it when I first tested. This PR can be closed since the env var actually does take effect.

Reply:
All right, thank you for trying again and the contribution.  I'll close this.